source/codegen/metadata/nidmm/enums.py

```diff
@@ -150,6 +159,7 @@
         ]
     },
     'CompensatedOhms': {
```

Comment:
Should probably just rely on the new OffsetCompensatedOhms that was coming from base metadata with the same values.